src/components/githubProfileCard/GithubProfileCard.js

```diff
@@ -25,11 +25,12 @@ export default function GithubProfileCard({prof}) {
               <div className="location-div">
                 <span className="desc-prof">
                   <svg
-                    viewBox="0 0 12 16"
+                    viewBox="-0.5 -2 20 19"
```

Comment:
Just tried to find symmetry and height size with the text beside the location icon. Checked if the newly updated size matches that of the text with the naked eye.